account_due_list_payment_mode/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,4 @@
+* Obertix Free Solutions <http://obertix.net>
+* Sergio Teruel <sergio.teruel@tecnativa.com>
+* Albert De La Fuente <info@haevas.com>
+* Vicent Cubells <vicent.cubells@tecnativa.com>
```

Comment:
Add yourself here

Reply:
why? it's necessary? my contribution it's just a simple migration, only changing version number. 🤔 But ok.

Reply:
Well, if you don't want... but I think each contribution deserves to be listed.

Reply:
ok, I included myself.